doc/UsersGuide/source/BuildingRunningTesting/index.rst

```diff
@@ -0,0 +1,11 @@
+Building, Running, and Testing HAFS
+========================================================
+
+.. toctree::
+   :maxdepth: 3
+
+   ExtQuickStart
+   GitHubBasicSteps
+   QuickStart
+   RegressionTest
+   HAFSCodeContributionGuidelines
```

Comment:
@jdkublnick Please place QuickStart first, followed by ExtQuickStart, and then RegressionTest. 
After some thought, I think it's best to create a separate "Contributor's Guide" section where `GitHubBasicSteps.rst` and `HAFSCodeContributionGuidelines.rst` can live.